### PR TITLE
feat: add custom icon boilerplate; add iconify library

### PR DESCRIPTION
**Pull request by Aztriltus**
# Changes
- Add CustomIcon boilerplate for creating custom icons
- Add Iconify react library for other icons used in Figma 
- Added README for both

# Screenshots

CleanShot 2023-09-11 at 23 54 41@2x

### CODE REVIEW

File: src/pages/index.tsx [hunk 1]
```diff
@@ -4,6 +4,9 @@ import { APP_THEMES } from "@/common/tools/tailwind/themes/appTheme";
 import { useTheme } from "next-themes";
 import { useState, useEffect, useCallback } from "react";
 import { PageHead } from "@/common/components/PageHead";
+import { Icon } from "@iconify-icon/react";
+import { CrossIcon } from "@/common/components/CustomIcon/CrossIcon";
```
Comment (jeromeandrewong) [inline src/pages/index.tsx:8]: Think you forgot to remove `CrossIcon` here as it has not been defined

Comment (Aztriltus) [reply src/pages/index.tsx:8]: Oh right, thanks for checking that!

**Review by jeromeandrewong — DISMISSED** [review]
LGTM 🛳️🛳️

**Review by DavidLHW — APPROVED** [review]
CLGTM @ [d84e73d](https://github.com/AfterClass-io/afterclass.io-v2/pull/15/commits/d84e73def1f60cca47776c70f0559e2981a261f3)

**Pull request merged**